### PR TITLE
Add identify-cluster

### PR DESCRIPTION
**Pull request by bhaskar-apple**
#### Problem
<the issue this PR is intended to address>

 #### Summary of Changes
<what's in this PR>

fixes #<issue number, or numbers. if no issue, please create one>

### CODE REVIEW

File: src/app/util/util.c [hunk 1]
```diff
@@ -1400,3 +1400,19 @@ uint8_t emberAfMake8bitEncodedChanPg(uint8_t page, uint8_t channel)
         return channel | ENCODED_8BIT_CHANPG_PAGE_MASK_PAGE_0;
     }
 }
+
+EmberStatus emberAfScheduleServerTick(uint8_t endpoint, EmberAfClusterId clusterId, uint32_t delayMs)
+{
+    return EMBER_SUCCESS; // Stub for now.
```
Comment (bhaskar-apple) [inline src/app/util/util.c:1406]: @rwalker-apple : do you have an issue tracking the work to moving silabs onto chip timers
These functions here need to be properly implemented.

**Comment by rwalker-apple** [comment]
#658 is the closest

File: src/app/util/util.c (hunk 1 above)
Comment (rwalker-apple) [reply src/app/util/util.c:1406]: project-chip#658 is the closest

File: examples/wifi-echo/server/esp32/main/gen/clusters-callback-stubs.c [hunk 2]
```diff
@@ -722,20 +745,20 @@ bool emberAfIasZoneClusterZoneStatusChangeNotificationCallback(uint16_t zoneStat
  *
  * @param identifyTime   Ver.: always
  */
-bool emberAfIdentifyClusterIdentifyCallback(uint16_t identifyTime)
-{
-    return false;
-}
+// bool emberAfIdentifyClusterIdentifyCallback(uint16_t identifyTime)
```
Comment (rwalker-apple) [inline examples/wifi-echo/server/esp32/main/gen/clusters-callback-stubs.c:748]: just delete these, I think

**Pull request closed without merging**